Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV RUBY_MAJOR_VERSION 2.6.0
 ENV RUBY_VERSION 2.6.3
 ENV BUNDLER_VERSION 2.0.1
```

Comment:
Please update bundler to 2.0.2 (https://devcenter.heroku.com/changelog-items/1656)